### PR TITLE
Edited Player::Level

### PR DESCRIPTION
**Pull request by NaoUnderscore**

### CODE REVIEW

**Review by iopietro — CHANGES_REQUESTED** [review]
Switch to dev branch

**Comment by NaoUnderscore** [comment]
My bad, I was PR'ing too much and I didn't see that.

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -751,9 +751,7 @@ public byte Level
                 if (ReferenceHub.scp079PlayerScript == null || ReferenceHub.scp079PlayerScript.Lvl == value)
                     return;
 
-                ReferenceHub.scp079PlayerScript.Lvl = value;
-
-                ReferenceHub.scp079PlayerScript.TargetLevelChanged(Connection, value);
+                ReferenceHub.scp079PlayerScript.ForceLevel(value, false);
```
Comment (iopietro) [inline Exiled.API/Features/Player.cs:754]:
```suggestion
                ReferenceHub.scp079PlayerScript.ForceLevel(value, true);
```

File: Exiled.API/Features/Player.cs [hunk 2]
```diff
@@ -1090,6 +1088,20 @@ public static Player Get(string args)
         /// <param name="camera">The <see cref="Camera079"/> object to switch to.</param>
         public void SetCamera(Camera079 camera) => SetCamera(camera.cameraId);
 
+        /// <summary>
+        /// Sets the player's level.
+        /// Only applies if the player is SCP-079.
+        /// </summary>
+        /// <param name="newLevel">The level to be set.</param>
+        /// <param name="displayNewLevel">Indicates whether the message should be displayed.</param>
+        public void SetLevel(byte newLevel, bool displayMessage)
```
Comment (iopietro) [inline Exiled.API/Features/Player.cs:1097]: This method does the same thing as the `Level` property, except for `displayMessage`, it's almost like a duplicate, I don't think that's needed.

**Review by iopietro — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Player.cs [hunk 3]
```diff
@@ -8,7 +8,7 @@
 namespace Exiled.API.Features
 {
     using System;
-    using System.Collections.Generic;
+    using System.Collections.Generic;https://www.animeworld.tv/play/fumetsu-no-anata-e.tY9Ls/Tf-pb2
```
Comment (joker-119) [inline Exiled.API/Features/Player.cs:11]: wtf is this?

**Review by joker-119 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Player.cs (hunk 3 above)
Comment (NaoUnderscore) [reply Exiled.API/Features/Player.cs:11]: Uhm.. Joker I'm sorry, really. lol

Comment (PintTheDragon) [reply Exiled.API/Features/Player.cs]: animeworld.tv

**Review by iopietro — APPROVED** [review]

**Review by joker-119 — APPROVED** [review]

**Pull request merged**